### PR TITLE
[1.20] bump containers/image to v5.10.6

### PR DESCRIPTION
**Pull request by haircommander**
to pick up a fd leak

Signed-off-by: Peter Hunt <pehunt@redhat.com>

#### What type of PR is this?

> /kind api-change
/kind bug
> /kind ci
> /kind cleanup
> /kind dependency-change
> /kind deprecation
> /kind design
> /kind documentation
> /kind failing-test
> /kind feature
> /kind flake
> /kind other

#### Does this PR introduce a user-facing change?

```release-note
fix a couple of FD leaks when pulling images
```

### CODE REVIEW

**Review by mrunalp — APPROVED** [review]
/lgtm

File: contrib/test/integration/build/cri-tools.yml [hunk 1]
```diff
@@ -4,7 +4,7 @@
   git:
     repo: "https://github.com/kubernetes-sigs/cri-tools.git"
     dest: "{{ ansible_env.GOPATH }}/src/github.com/kubernetes-sigs/cri-tools"
-    version: v1.20.0
+    version: v1.21.0
```
Comment (mrunalp) [inline contrib/test/integration/build/cri-tools.yml:7]: Why bump to 1.21 for release-1.20?

Comment (haircommander) [reply contrib/test/integration/build/cri-tools.yml:7]: to get https://github.com/kubernetes-sigs/cri-tools/pull/733 easier :)

**Pull request merged**